fineract-provider/src/main/java/org/apache/fineract/infrastructure/documentmanagement/api/ImagesApiResource.java

```diff
@@ -140,7 +140,8 @@ public Response retrieveImage(@PathParam("entity") final String entityName, @Pat
             imageDataURISuffix = ContentRepositoryUtils.ImageDataURIsuffix.PNG.getValue();
         }
 
-        final String clientImageAsBase64Text = imageDataURISuffix + Base64.encodeBytes(imageData.getContentOfSize(maxWidth, maxHeight));
+        byte[] resizedImage = imageData.getContentOfSize(maxWidth, maxHeight);
+        final String clientImageAsBase64Text = imageDataURISuffix + Base64.getEncoder().encode(resizedImage);
```

Comment:
```suggestion
        final String clientImageAsBase64Text = imageDataURISuffix + Base64.getEncoder().encodeToString(resizedImage);
```

I suggest you use this instead. Check this for reference: https://www.baeldung.com/java-base64-encode-and-decode.
Should be able to merge this once that is done.

Reply:
done! Thanks you, very much.